packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/utils/useEditDialog.ts

```diff
@@ -124,12 +118,11 @@ export default function openEditDialog(editor: any, markupToHtml: any, dispatchD
 					label: 'Language',
 					// Katex is a special case with special opening/closing tags
 					// and we don't currently handle switching the language in this case.
-					disabled: source.language === 'katex',
+					enabled: source.language !== 'katex',
 				},
 				{
 					type: 'textarea',
 					name: 'codeTextArea',
-					value: source.content,
```

Comment:
Changed after replacing an `any` type with a stronger type: `value` was giving a type error. Removing `value` does not prevent the edit dialog from opening with an existing code block's content.